modules/azure/common.go

```diff
@@ -52,3 +52,19 @@ func getTargetAzureResourceGroupName(resourceGroupName string) (string, error) {
 
 	return resourceGroupName, nil
 }
+
+// safePtrToString converts a string pointer to a non-pointer string value, or to "" if the pointer is nil.
+func safePtrToString(raw *string) string {
```

Comment:
add unit test for these methods to common_test.go

Reply:
Done